src/File.php

```diff
@@ -121,17 +121,16 @@ class File extends DataObject implements ShortcodeHandler, AssetContainer, Thumb
     );
 
     private static $has_one = array(
-        "Parent" => "SilverStripe\\Assets\\File",
-        "Owner" => "SilverStripe\\Security\\Member"
+        "Parent" => File::class,
+        "Owner" => Member::class,
     );
 
     private static $defaults = array(
         "ShowInSearch" => 1,
     );
 
     private static $extensions = array(
-        "SilverStripe\\ORM\\Hierarchy\\Hierarchy",
-        "SilverStripe\\ORM\\Versioning\\Versioned"
+        Hierarchy::class,
```

Comment:
How is Versioned added to File? It's important that this is on by default in our default installs.